### ISSUE TITLE
How to use A100 GPU to train the data?

### ISSUE THREAD

**Issue by LemingVan** [alenzenx/TrackNetV3#8]
Hello @alenzenx ,

我在使用colab上80G顯卡訓練時，使用`python train.py --num_frame 3 --epochs 30 --batch_size 24 --learning_rate 0.001 --save_dir exp`報錯，大致意思是get_num=0，請問你知道怎麼發揮大顯卡的優勢，讓訓練的時間減短嗎？

When I was using the 80G graphics card for training on Colab, running `python train.py --num_frame 3 --epochs 30 --batch_size 24 --learning_rate 0.001 --save_dir exp` caused an error, roughly meaning get_num=0. Do you know how to take advantage of a powerful graphics card to reduce training time?

**Comment by alenzenx** [comment]
--batch_size 加大是最好的方法

**Comment by LemingVan** [comment]
感謝回覆，我想再問兩個問題：

（1）當使用`train.py`訓練10個epoch暫停了，能通過熱啟動從第11個epoch接著訓練嗎？該如何實現？
（2）在創造數據集時，請問`Chancel label of the ball`記錄的是什麼信息？

**Comment by alenzenx** [comment]
（1）記得先把 model_last.pt 存起來備用，它是上一個 epoch 的權重，使用 --resume_training 可以恢復訓練。
（2）Chancel label of the ball 應該是清除那張圖被標註的點。

**Comment by LemingVan** [comment]
好的，非常感謝！
